templates/email/bexley/waste/other-reported.html

```diff
@@ -30,6 +32,16 @@ <h1 style="[% h1_style %]">Your report has been&nbsp;logged</h1>
       <br><br>
       If you need to contact us about this report, please quote your reference number.
     </p>
+  [% ELSIF is_container_request %]
+    <p style="[% p_style %]">
+      Your request has been sent to our waste contractor and will be completed within 10 working days.
```

Comment:
If you use `[% cobrand.wasteworks_config.request_timeframe || '10 working days' %]` here, as e.g. the default other-reported.html does, we can more easily override the timeframe should they change it (other councils have sometimes, if they're suffering from delays/shortages etc).

Reply:
This text should presumably also be added to the .txt version as well